### ISSUE TITLE
xfxentry marign

### ISSUE THREAD

**Issue by brux88** [XamFormsExtended/Xfx.Controls#46]
its possible set margin from label and entry, in ios is too small

# Feature Request:
Please fill in what you would like

**Comment by ChaseFlorell** [comment]
at this time it's not possible. we've got two options to solve this for you.

1) would be to fork the repo and change the iOS platform to your liking
2) would be to flesh out what it would look like to make it settable cross platform and submit a PR

in the meantime, I'm going to adjust the naming a little on this issue and leave it open. If other's pipe up and say that the spacing on iOS is too small, maybe we can just increase that spacing overall.

**Comment by ChaseFlorell** [comment]
does this also include the spacing between the entry and the error text below?

**Comment by brux88** [comment]
no only include spacing between entry and label

**Comment by Nishita1121** [comment]
Yes, spacing on iOS is too small. So, what is the solution to increase the spacing b/w xfxEntry and it's floating labels.

**Comment by Itouqeer** [comment]
Yes spacing on iOS is too small. Please fix. #brux88 do you found any solution?

**Comment by ChaseFlorell** [comment]
I do take pull requests.
https://github.com/XamFormsExtended/Xfx.Controls/blob/master/src/Xfx.Controls.iOS/Controls/FloatLabeledTextField.cs

**Comment by Itouqeer** [comment]
@ChaseFlorell thanks for your swift response. Dude i am a beginner can you please guide how i can solve this?

**Comment by ChaseFlorell** [comment]
The spacing is hard-coded (and will remain so). 
I looked at the spacing on Android, and agree that iOS needs an additional gap, The file I linked to above is where you will adjust the spacing.

**Comment by Itouqeer** [comment]
@ChaseFlorell dude i just installed the package uaing nuget how can i edit that?

**Comment by ChaseFlorell** [comment]
you'd have to fork this repository, make the adjustments, and submit a pull request.
If you're not comfortable with that, you'll have to wait until I can tackle it, or someone else is willing to take it on.

**Comment by Itouqeer** [comment]
Ok @ChaseFlorell thanks for your time. Ok i can wait. Hope it can be solved soon.

**Comment by technololy** [comment]
Hi Guys. This solution would really be a boost for this plugin. Thanks
